domain/secret/service/delete.go

```diff
@@ -35,7 +36,17 @@ func (s *SecretService) DeleteSecret(ctx context.Context, uri *secrets.URI, para
 	if err := s.canManage(ctx, uri, params.Accessor, params.LeaderToken); err != nil {
 		return errors.Trace(err)
 	}
+	return s.secretState.RunAtomic(ctx, func(ctx domain.AtomicContext) error {
+		return s.InternalDeleteSecret(ctx, uri, params)
+	})
+}
 
+// InternalDeleteSecret removes the specified secret.
+// If revisions is nil or the last remaining revisions are removed.
+// It returns [secreterrors.PermissionDenied] if the secret cannot be managed by the accessor.
```

Comment:
This line and the last line of the comment, which one is correct?

Reply:
Ah, copy / paste error.
`It returns [secreterrors.PermissionDenied] if the secret cannot be managed by the accessor.` should not be there. 
Plus there's a typo in the line about revisions which was copied from the DeleteSecret method above so both need to be fixed.